generators/gae/templates/pom-plugin.xml.ejs

```diff
@@ -1,3 +1,21 @@
+<%#
+ Copyright 2013-2019 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
                 <configuration>
                     <deploy.project><%=gcpProjectId%></deploy.project>
                     <deploy.version>1</deploy.version>
```

Comment:
would it be possible to use ${project.version} ?

Reply:
@saturnism : No we cannot; because at it's current form; the `${project.version}` is set to `0.0.1-SNAPSHOT` and GAE does not accept this versioning scheme. Probably this is why it was set this way to begin with. The error we get would be; 

`ERROR: (gcloud.app.deploy) argument --version/-v: Bad value [0.0.1-SNAPSHOT]: May only contain lowercase letters, digits, and hyphens. Must begin and end with a letter or digit. Must not exceed 63 characters.
`